src/components/columns/columns.js

```diff
@@ -8,7 +8,7 @@ import modifiers from '../../modifiers';
 import Element from '../element';
 
 const breakpoints = [null].concat(
-  Object.keys(CONSTANTS.BREAKPOINTS).map(key => CONSTANTS.BREAKPOINTS[key]),
+  Object.keys(CONSTANTS.BREAKPOINTS).map((key) => CONSTANTS.BREAKPOINTS[key]),
```

Comment:
using `Object.values(CONSTANTS.BREAKPOINTS)` would be simpler here.

Reply:
I totally forgot about this. this line change on the PR because the prettier change the rules. I will change it